filter_plugins/opg_custom.py

```diff
@@ -113,6 +113,24 @@ def find_rds_instance(rds_list, db_name):
     return db_info
 
 
+def merge_config_dictionaries(*dicts):
+    """
+    Merges n dictionaries of configuration data
+    :param dicts:
+    :return:
+    """
+    res_dict = {}
+
+    if len(dicts) <= 0:
+        return res_dict
```

Comment:
you might be masking errors during the invocation of this plugin, as you could be passing ```None``` and returning ```{ }``.

And I suspect there wouldn't many instances where you'd want to pass ```None``` here